frontend/core-ui/src/modules/notification/my-inbox/components/MyInboxNavigationItem.tsx

```
@@ -1,18 +1,56 @@
 import { useUnreadNotificationCount } from '@/notification/my-inbox/hooks/useUnreadNotificationCount';
 import { IconInbox } from '@tabler/icons-react';
+import {
+  NavigationMenuLinkItem,
+  Skeleton,
+  Button,
+  DropdownMenu,
+} from 'erxes-ui';
 import { Badge, NavigationMenuLinkItem, Skeleton } from 'erxes-ui';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'NavigationMenuLinkItem'. Consider to delete it or rename it.

'NavigationMenuLinkItem' is defined here:


(lint/suspicious/noRedeclare)